bridge/server/app.js

```diff
@@ -18,6 +17,10 @@ let apiToken = process.env.API_TOKEN;
 let cliDownloadLink = process.env.CLI_DOWNLOAD_LINK;
 let integrationsPageLink = process.env.INTEGRATIONS_PAGE_LINK;
 let lookAndFeelUrl = process.env.LOOK_AND_FEEL_URL;
+const throttleBucket /* {[ip: string]: number[]} */ = {};
+const requestTimeLimit = 60 * 60 * 1000; // 1 hour
+const requestLimitWithinTime = 10; // 10 requests within {requestTimeLimit}
+const cleanBucketsTimeout = 60 * 60 * 1000; // clean buckets every 1 hour
```

Comment:
Can we make those parameters configurable via environment variables and use these values as defaults?
I would use `minutes` as unit for the `requestTimeLimit` and `cleanBucketsTimeout` parameter.